CHANGELOG.md

```diff
@@ -1,5 +1,61 @@
 # Changelog
 
+## [v5.1.0](https://github.com/nextcloud/eslint-config/tree/v5.1.0) (2021-05-03)
```

Comment:
What about 5.1.1 ? xD

Reply:
manually?

Reply:
Sure :+1: 
It's just a fix for changelog and stuff ;)